### ISSUE TITLE
Error running demo (Tf has two or more unconnected trees)

### ISSUE THREAD

**Issue by gribes02** [open-navigation/opennav_coverage#76]
## Description
I encountered an error when running the path_coverage_demo. There are 2 trees as can be seen in the picture below. I have not changed anything in the launch file nor source code. I was assuming the demo would work as is.

## Steps to reproduce error
1. Built Fields2Cover version 1.2.1
2. Cloned opennav_coverage repository
3. `colcon build --symlink-install && source install/setup.bash`
4. `ros2 launch opennav_coverage_demo coverage_demo_launch.py`

## Error
```
[component_container_isolated-5] [INFO] [1727081091.970542309] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
```
![image](https://github.com/user-attachments/assets/1c879730-14cd-4378-9576-248534bcfb83)

Any help on this would be highly appreciated!

**Comment by SteveMacenski** [comment]
Are there any errors from the simulator or otherwise? The simulator is what provides the odom->base_footprint transformation

**Comment by gribes02** [comment]
Hi @SteveMacenski,

When I tried running the launch file now I get a different error. the TF tree still looks the same as when I sent the picture however the result of `get_state` is inactive and it does that the whole time it is running and I do not get the `Timed out waiting for transform from base_link to odom to become available` warning. I also see that there are some ALSA errors where it can not find some files or directories, but not sure if that is what is impeding me from running the demo. 

```
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [demo_coverage-7]: process started with pid [40623]
[INFO] [gzserver-1]: process started with pid [40611]
[INFO] [robot_state_publisher-2]: process started with pid [40613]
[INFO] [spawn_entity.py-3]: process started with pid [40615]
[INFO] [rviz2-4]: process started with pid [40617]
[INFO] [component_container_isolated-5]: process started with pid [40619]
[INFO] [static_transform_publisher-6]: process started with pid [40621]
[static_transform_publisher-6] [WARN] [1727164367.716792261] []: Old-style arguments are deprecated; see --help for new-style arguments
[static_transform_publisher-6] [INFO] [1727164367.725734948] [static_transform_publisher_LelRiMGpPDBpe64C]: Spinning until stopped - publishing transform
[static_transform_publisher-6] translation: ('0.000000', '0.000000', '0.000000')
[static_transform_publisher-6] rotation: ('0.000000', '0.000000', '0.000000', '1.000000')
[static_transform_publisher-6] from 'map' to 'odom'
[robot_state_publisher-2] [INFO] [1727164367.726680712] [robot_state_publisher]: got segment base_footprint
[robot_state_publisher-2] [INFO] [1727164367.726743904] [robot_state_publisher]: got segment base_link
[robot_state_publisher-2] [INFO] [1727164367.726749772] [robot_state_publisher]: got segment base_scan
[robot_state_publisher-2] [INFO] [1727164367.726753085] [robot_state_publisher]: got segment camera_depth_frame
[robot_state_publisher-2] [INFO] [1727164367.726755981] [robot_state_publisher]: got segment camera_depth_optical_frame
[robot_state_publisher-2] [INFO] [1727164367.726759234] [robot_state_publisher]: got segment camera_link
[robot_state_publisher-2] [INFO] [1727164367.726762184] [robot_state_publisher]: got segment camera_rgb_frame
[robot_state_publisher-2] [INFO] [1727164367.726765020] [robot_state_publisher]: got segment camera_rgb_optical_frame
[robot_state_publisher-2] [INFO] [1727164367.726767980] [robot_state_publisher]: got segment caster_back_left_link
[robot_state_publisher-2] [INFO] [1727164367.726770848] [robot_state_publisher]: got segment caster_back_right_link
[robot_state_publisher-2] [INFO] [1727164367.726773512] [robot_state_publisher]: got segment imu_link
[robot_state_publisher-2] [INFO] [1727164367.726776496] [robot_state_publisher]: got segment wheel_left_link
[robot_state_publisher-2] [INFO] [1727164367.726779339] [robot_state_publisher]: got segment wheel_right_link
[rviz2-4] QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-eutrob'
[demo_coverage-7] Waiting for bt_navigator to become active..
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[rviz2-4] [INFO] [1727164367.933739114] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-4] [INFO] [1727164367.933812790] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[rviz2-4] [INFO] [1727164367.946204918] [rviz2]: Stereo is NOT SUPPORTED
[spawn_entity.py-3] [INFO] [1727164367.963272709] [spawn_entity]: Spawn Entity started
[spawn_entity.py-3] [INFO] [1727164367.963503474] [spawn_entity]: Loading entity XML from file /opt/ros/humble/share/nav2_bringup/worlds/waffle.model
[spawn_entity.py-3] [INFO] [1727164367.963985957] [spawn_entity]: Waiting for service /spawn_entity, timeout = 30
[spawn_entity.py-3] [INFO] [1727164367.964172384] [spawn_entity]: Waiting for service /spawn_entity
[gzserver-1] ALSA lib confmisc.c:855:(parse_card) cannot find card '0'
[gzserver-1] ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_card_inum returned error: No such file or directory
[gzserver-1] ALSA lib confmisc.c:422:(snd_func_concat) error evaluating strings
[gzserver-1] ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
[gzserver-1] ALSA lib confmisc.c:1334:(snd_func_refer) error evaluating name
[gzserver-1] ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
[gzserver-1] ALSA lib conf.c:5701:(snd_config_expand) Evaluate error: No such file or directory
[gzserver-1] ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM default
[gzserver-1] AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
[spawn_entity.py-3] [INFO] [1727164368.469523073] [spawn_entity]: Calling service /spawn_entity
[spawn_entity.py-3] [INFO] [1727164368.739018622] [spawn_entity]: Spawn status: SpawnEntity: Successfully spawned entity [tb3]
[INFO] [spawn_entity.py-3]: process has finished cleanly [pid 40615]
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive

```

Thank you for the help!

**Comment by gribes02** [comment]
Hi @SteveMacenski!

This is the full message when running `ros2 launch opennav_coverage_demo coverage_demo_launch.py`. 

I believe I am getting some ALSA error messages where it can not find some files or directories. Would that be the reason why there are some issues with the simulation?


```
[INFO] [component_container_isolated-5]: process started with pid [182]
[INFO] [static_transform_publisher-6]: process started with pid [184]
[static_transform_publisher-6] [WARN] [1727166966.473157426] []: Old-style arguments are deprecated; see --help for new-style arguments
[static_transform_publisher-6] [INFO] [1727166966.487260568] [static_transform_publisher_c4OodO8Ja2nLAnha]: Spinning until stopped - publishing transform
[static_transform_publisher-6] translation: ('0.000000', '0.000000', '0.000000')
[static_transform_publisher-6] rotation: ('0.000000', '0.000000', '0.000000', '1.000000')
[static_transform_publisher-6] from 'map' to 'odom'
[robot_state_publisher-2] [INFO] [1727166966.490474386] [robot_state_publisher]: got segment base_footprint
[robot_state_publisher-2] [INFO] [1727166966.490542562] [robot_state_publisher]: got segment base_link
[robot_state_publisher-2] [INFO] [1727166966.490548617] [robot_state_publisher]: got segment base_scan
[robot_state_publisher-2] [INFO] [1727166966.490551659] [robot_state_publisher]: got segment camera_depth_frame
[robot_state_publisher-2] [INFO] [1727166966.490554376] [robot_state_publisher]: got segment camera_depth_optical_frame
[robot_state_publisher-2] [INFO] [1727166966.490557346] [robot_state_publisher]: got segment camera_link
[robot_state_publisher-2] [INFO] [1727166966.490560038] [robot_state_publisher]: got segment camera_rgb_frame
[robot_state_publisher-2] [INFO] [1727166966.490562618] [robot_state_publisher]: got segment camera_rgb_optical_frame
[robot_state_publisher-2] [INFO] [1727166966.490565341] [robot_state_publisher]: got segment caster_back_left_link
[robot_state_publisher-2] [INFO] [1727166966.490567819] [robot_state_publisher]: got segment caster_back_right_link
[robot_state_publisher-2] [INFO] [1727166966.490570231] [robot_state_publisher]: got segment imu_link
[robot_state_publisher-2] [INFO] [1727166966.490572895] [robot_state_publisher]: got segment wheel_left_link
[robot_state_publisher-2] [INFO] [1727166966.490575712] [robot_state_publisher]: got segment wheel_right_link
[rviz2-4] QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-eutrob'
[component_container_isolated-5] [INFO] [1727166966.712123670] [nav2_container]: Load Library: /opt/ros/humble/lib/libcontroller_server_core.so
[component_container_isolated-5] [INFO] [1727166966.725211440] [nav2_container]: Found class: rclcpp_components::NodeFactoryTemplate<nav2_controller::ControllerServer>
[component_container_isolated-5] [INFO] [1727166966.725252168] [nav2_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<nav2_controller::ControllerServer>
[component_container_isolated-5] [INFO] [1727166966.730710044] [controller_server]: 
[component_container_isolated-5] 	controller_server lifecycle node launched. 
[component_container_isolated-5] 	Waiting on external lifecycle transitions to activate
[component_container_isolated-5] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[spawn_entity.py-3] [INFO] [1727166966.734118435] [spawn_entity]: Spawn Entity started
[spawn_entity.py-3] [INFO] [1727166966.734594135] [spawn_entity]: Loading entity XML from file /opt/ros/humble/share/nav2_bringup/worlds/waffle.model
[spawn_entity.py-3] [INFO] [1727166966.736432414] [spawn_entity]: Waiting for service /spawn_entity, timeout = 30
[spawn_entity.py-3] [INFO] [1727166966.736824410] [spawn_entity]: Waiting for service /spawn_entity
[component_container_isolated-5] [INFO] [1727166966.742320980] [controller_server]: Creating controller server
[demo_coverage-7] Waiting for bt_navigator to become active..
[component_container_isolated-5] [INFO] [1727166966.750356231] [local_costmap.local_costmap]: 
[component_container_isolated-5] 	local_costmap lifecycle node launched. 
[component_container_isolated-5] 	Waiting on external lifecycle transitions to activate
[component_container_isolated-5] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[component_container_isolated-5] [INFO] [1727166966.753050940] [local_costmap.local_costmap]: Creating Costmap
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/controller_server' in container 'nav2_container'
[component_container_isolated-5] [INFO] [1727166966.756051945] [nav2_container]: Load Library: /home/eutrob/open_navigation_ws/install/opennav_coverage/lib/libopennav_coverage_core.so
[component_container_isolated-5] [INFO] [1727166966.807960291] [nav2_container]: Found class: rclcpp_components::NodeFactoryTemplate<opennav_coverage::CoverageServer>
[component_container_isolated-5] [INFO] [1727166966.808017913] [nav2_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<opennav_coverage::CoverageServer>
[component_container_isolated-5] [INFO] [1727166966.815395859] [coverage_server]: 
[component_container_isolated-5] 	coverage_server lifecycle node launched. 
[component_container_isolated-5] 	Waiting on external lifecycle transitions to activate
[component_container_isolated-5] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[component_container_isolated-5] [INFO] [1727166966.815772937] [coverage_server]: Creating coverage_server
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/coverage_server' in container 'nav2_container'
[component_container_isolated-5] [INFO] [1727166966.819305003] [nav2_container]: Load Library: /home/eutrob/open_navigation_ws/install/backported_bt_navigator/lib/libbt_navigator_core.so
[component_container_isolated-5] [INFO] [1727166966.825940556] [nav2_container]: Found class: rclcpp_components::NodeFactoryTemplate<backported_bt_navigator::BtNavigator>
[component_container_isolated-5] [INFO] [1727166966.825990158] [nav2_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<backported_bt_navigator::BtNavigator>
[component_container_isolated-5] [INFO] [1727166966.829055810] [bt_navigator]: 
[component_container_isolated-5] 	bt_navigator lifecycle node launched. 
[component_container_isolated-5] 	Waiting on external lifecycle transitions to activate
[component_container_isolated-5] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[component_container_isolated-5] [INFO] [1727166966.833135654] [bt_navigator]: Creating
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/bt_navigator' in container 'nav2_container'
[component_container_isolated-5] [INFO] [1727166966.836853206] [nav2_container]: Load Library: /opt/ros/humble/lib/libvelocity_smoother_core.so
[component_container_isolated-5] [INFO] [1727166966.838775036] [nav2_container]: Found class: rclcpp_components::NodeFactoryTemplate<nav2_velocity_smoother::VelocitySmoother>
[component_container_isolated-5] [INFO] [1727166966.838810202] [nav2_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<nav2_velocity_smoother::VelocitySmoother>
[component_container_isolated-5] [INFO] [1727166966.843297594] [velocity_smoother]: 
[component_container_isolated-5] 	velocity_smoother lifecycle node launched. 
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/velocity_smoother' in container 'nav2_container'
[component_container_isolated-5] 	Waiting on external lifecycle transitions to activate
[component_container_isolated-5] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[component_container_isolated-5] [INFO] [1727166966.845919617] [nav2_container]: Load Library: /opt/ros/humble/lib/libnav2_lifecycle_manager_core.so
[component_container_isolated-5] [INFO] [1727166966.847616690] [nav2_container]: Found class: rclcpp_components::NodeFactoryTemplate<nav2_lifecycle_manager::LifecycleManager>
[component_container_isolated-5] [INFO] [1727166966.847640513] [nav2_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<nav2_lifecycle_manager::LifecycleManager>
[component_container_isolated-5] [INFO] [1727166966.852468732] [lifecycle_manager_navigation]: Creating
[component_container_isolated-5] [INFO] [1727166966.855364473] [lifecycle_manager_navigation]: Creating and initializing lifecycle service clients
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/lifecycle_manager_navigation' in container 'nav2_container'
[component_container_isolated-5] [INFO] [1727166966.856959999] [lifecycle_manager_navigation]: Starting managed nodes bringup...
[component_container_isolated-5] [INFO] [1727166966.856995900] [lifecycle_manager_navigation]: Configuring controller_server
[component_container_isolated-5] [INFO] [1727166966.857075640] [controller_server]: Configuring controller interface
[component_container_isolated-5] [INFO] [1727166966.857162245] [controller_server]: getting goal checker plugins..
[component_container_isolated-5] [INFO] [1727166966.857261701] [controller_server]: Controller frequency set to 20.0000Hz
[component_container_isolated-5] [INFO] [1727166966.857575424] [local_costmap.local_costmap]: Configuring
[component_container_isolated-5] [INFO] [1727166966.861895461] [local_costmap.local_costmap]: Using plugin "voxel_layer"
[component_container_isolated-5] [INFO] [1727166966.868451405] [local_costmap.local_costmap]: Subscribed to Topics: scan
[component_container_isolated-5] [INFO] [1727166966.869959775] [local_costmap.local_costmap]: Initialized plugin "voxel_layer"
[component_container_isolated-5] [INFO] [1727166966.869986636] [local_costmap.local_costmap]: Using plugin "inflation_layer"
[component_container_isolated-5] [INFO] [1727166966.870470527] [local_costmap.local_costmap]: Initialized plugin "inflation_layer"
[component_container_isolated-5] [INFO] [1727166966.874520251] [controller_server]: Created progress_checker : progress_checker of type nav2_controller::SimpleProgressChecker
[component_container_isolated-5] [INFO] [1727166966.875494345] [controller_server]: Created goal checker : general_goal_checker of type nav2_controller::SimpleGoalChecker
[component_container_isolated-5] [INFO] [1727166966.875720629] [controller_server]: Controller Server has general_goal_checker  goal checkers available.
[component_container_isolated-5] [INFO] [1727166966.876847217] [controller_server]: Created controller : FollowPath of type nav2_regulated_pure_pursuit_controller::RegulatedPurePursuitController
[component_container_isolated-5] [INFO] [1727166966.878812171] [controller_server]: Controller Server has FollowPath  controllers available.
[component_container_isolated-5] [INFO] [1727166966.881948567] [lifecycle_manager_navigation]: Configuring bt_navigator
[component_container_isolated-5] [INFO] [1727166966.882017068] [bt_navigator]: Configuring
[component_container_isolated-5] [INFO] [1727166966.883723164] [bt_navigator]: Creating navigator id navigate_complete_coverage of type opennav_coverage_navigator/CoverageNavigator
[component_container_isolated-5] [INFO] [1727166966.911201370] [lifecycle_manager_navigation]: Configuring velocity_smoother
[component_container_isolated-5] [INFO] [1727166966.911330119] [velocity_smoother]: Configuring velocity smoother
[component_container_isolated-5] [INFO] [1727166966.912606037] [lifecycle_manager_navigation]: Configuring coverage_server
[component_container_isolated-5] [INFO] [1727166966.912679375] [coverage_server]: Configuring coverage_server
[component_container_isolated-5] [WARN] [1727166966.914431327] [coverage_server]: default_custom_order was not set! If using Custom Route mode, the custom order must be set per-request!
[component_container_isolated-5] [INFO] [1727166966.917112031] [lifecycle_manager_navigation]: Activating controller_server
[component_container_isolated-5] [INFO] [1727166966.917197523] [controller_server]: Activating
[component_container_isolated-5] [INFO] [1727166966.917236762] [local_costmap.local_costmap]: Activating
[component_container_isolated-5] [INFO] [1727166966.917254500] [local_costmap.local_costmap]: Checking transform
[component_container_isolated-5] [INFO] [1727166966.917287256] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[rviz2-4] [INFO] [1727166966.922018354] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-4] [INFO] [1727166966.922094358] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[rviz2-4] [INFO] [1727166966.940065874] [rviz2]: Stereo is NOT SUPPORTED
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[gzserver-1] ALSA lib confmisc.c:855:(parse_card) cannot find card '0'
[gzserver-1] ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_card_inum returned error: No such file or directory
[gzserver-1] ALSA lib confmisc.c:422:(snd_func_concat) error evaluating strings
[gzserver-1] ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
[gzserver-1] ALSA lib confmisc.c:1334:(snd_func_refer) error evaluating name
[gzserver-1] ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
[gzserver-1] ALSA lib conf.c:5701:(snd_config_expand) Evaluate error: No such file or directory
[gzserver-1] ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM default
[gzserver-1] AL lib: (EE) ALCplaybackAlsa_open: Could not open playback device 'default': No such file or directory
[component_container_isolated-5] [INFO] [1727166967.417463270] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[spawn_entity.py-3] [INFO] [1727166967.499003109] [spawn_entity]: Calling service /spawn_entity
[spawn_entity.py-3] [INFO] [1727166967.762800580] [spawn_entity]: Spawn status: SpawnEntity: Successfully spawned entity [tb3]
[component_container_isolated-5] [INFO] [1727166967.917621219] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[INFO] [spawn_entity.py-3]: process has finished cleanly [pid 178]
[component_container_isolated-5] [INFO] [1727166968.417407578] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[component_container_isolated-5] [INFO] [1727166968.917534036] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive
[component_container_isolated-5] [INFO] [1727166969.417384368] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[component_container_isolated-5] [INFO] [1727166969.917411480] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[component_container_isolated-5] [INFO] [1727166970.417417481] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[component_container_isolated-5] [INFO] [1727166970.917444505] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Could not find a connection between 'odom' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
[demo_coverage-7] Getting bt_navigator state...
[demo_coverage-7] Result of get_state: inactive

```

**Comment by SteveMacenski** [comment]
I can't say, that's probably something on your side or there's other logging that you're not displaying with gz server errors (or not waiting long enough for it to initialize). This seems to work on my side fine and no one else has reported this trouble.

**Comment by gribes02** [comment]
Yes it was just a matter of waiting, thanks for the help!
